browser/js/dashboard/dash.wrapper.controller.js

```diff
@@ -19,4 +26,9 @@ app.controller('DashWrapperCtrl', function($scope, $state, $mdDialog, MdHelpers)
         });
     };
 
+    $scope.$on('$stateChangeStart', function(){
+        $element.remove();
```

Comment:
Our state changes are (for some reason) not removing the old state from the
DOM until about a second after the transition. So I had to add these to
force clear the DOM before the new state loads (possibly because of one of
the animation modules we injected?? Idk. I troubleshooted this for a
whiiiile).

Chris Heller
781-718-7743
www.chrishellermusic.com

On Fri, Aug 19, 2016 at 3:28 PM, Omri Bernstein <notifications@github.com>
wrote:

> In browser/js/dashboard/dash.wrapper.controller.js
> <https://github.com/ctheller/snippets/pull/185#discussion_r75538928>:
>
> > @@ -19,4 +26,9 @@ app.controller('DashWrapperCtrl', function($scope, $state, $mdDialog, MdHelpers)
> >          });
> >      };
> >
> > +    $scope.$on('$stateChangeStart', function(){
> > +        $element.remove();
>
> Why is this needed?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/ctheller/snippets/pull/185/files/d48d0342aee8209110d6f7d9b9b55d06fe4d514d#r75538928>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/APJMPwbzzdJ_gGyqmMUc7uqocdA4Y99wks5qhgPagaJpZM4Jom5w>
> .
>